src/components/Header/Header.js

```diff
@@ -140,6 +140,8 @@ class Header extends Component {
 
       this.setState({ isAuthenticated: false });
       window.location.reload(false);
+    }else if(e === "2"){
```

Comment:
what is e ?

Reply:
e is parameter of function and e === "2" is second of user menu, that is, admin button